opencti-platform/opencti-graphql/src/domain/connector.ts

```diff
@@ -201,7 +222,27 @@ export const connectorTriggerUpdate = async (context: AuthContext, user: AuthUse
 // endregion
 
 // region syncs
-export const patchSync = async (context: AuthContext, user: AuthUser, id: string, patch: { runnning: boolean }) => {
+export const registerConnectorForIngestion = async (context: AuthContext, type: string, id: string, name: string, is_running: boolean) => {
```

Comment:
since we have 4 parameters here (type, id, name, is_running), I think it would be better to create a type and regroup them in one parameter (which will make sure we call the method with the right input)